pilot/pkg/xds/debug.go

```diff
@@ -330,9 +330,13 @@ func (s *DiscoveryServer) cachez(w http.ResponseWriter, req *http.Request) {
 		writeJSON(w, res)
 		return
 	}
-	keys := s.Cache.Keys()
-	sort.Strings(keys)
-	writeJSON(w, keys)
+	snapshot := s.Cache.Snapshot()
+	resources := make(map[string][]string) // Key is typeUrl and value is resource names.
+	for _, resource := range snapshot {
+		resourceType := resource.Resource.TypeUrl
+		resources[resourceType] = append(resources[resourceType], resource.Name)
```

Comment:
maybe `name/hash`? Seems the hash could be useful in some scenarios.. although not as useful as unobfuscated